spec/requests/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Users API', type: :request do
```

Comment:
Shouldn't this be `RSpec.describe Api::V1::UsersController` or something like that?

Reply:
I used Rspec request api not the controller Api which is the new recommend way to make integration tests.
Symmetrically request api isn't limited to controllers but rather to a bigger context like Users Api or Rooms Api where tests for the full request/response cycle lives.
Do you want me to use the subject notation?

Reply:
Yeah I'd keep the `type: :request` but for the sake of code cleanliness, I'd rather we separate tests by controller, just to keep things easy to manage

Reply:
Alright, Ill change it.

Reply:
Done